server/src/main/java/org/apache/druid/server/coordinator/stats/Stats.java

```diff
@@ -137,6 +137,10 @@ public static class CoordinatorRun
         = CoordinatorStat.toDebugAndEmit("dutyRunTime", "coordinator/time");
     public static final CoordinatorStat GROUP_RUN_TIME
         = CoordinatorStat.toDebugAndEmit("groupRunTime", "coordinator/global/time");
+    public static final CoordinatorStat CLONE_LOAD
+        = CoordinatorStat.toDebugAndEmit("cloneLoad", "coordinator/clone/load");
+    public static final CoordinatorStat CLONE_DROP
+        = CoordinatorStat.toDebugAndEmit("cloneDrop", "coordinator/clone/drop");
```

Comment:
These metrics should go under `Stats.Segments` to adhere to the scheme currently followed
- `segment/clone/assigned/count`
- `segment/clone/dropped/count`